types/mock-interceptor.d.ts

```diff
@@ -49,7 +49,7 @@ declare namespace MockInterceptor {
     /** Body to intercept on. */
     body?: string | RegExp | ((body: string) => boolean);
     /** Headers to intercept on. */
-    headers?: Record<string, string | RegExp | ((body: string) => boolean)>;
+    headers?: Record<string, string | RegExp | ((body: string) => boolean)> | ((headers: Record<string, string>) => boolean);
```

Comment:
Could you add an assertion for this type? We use tsd?

Reply:
Sure, where?

Reply:
test/types/mock-interceptor.test-d.ts